.lando.yml

```diff
@@ -4,13 +4,13 @@ config:
   php: '7.4'
   composer_version: '2.0.7'
   via: apache:2.4
-  database: mariadb:10.4
+  database: mariadb:10.5
   # https://docs.lando.dev/config/drupal9.html#using-drush
   # As of Drupal 9 and Drush 10 it is preferred you use a site-local install of Drush.
   # For that reason Lando will not globally install a version of Drush for Drupal 9 sites.
   drush: false
   # https://docs.lando.dev/config/drupal9.html#using-xdebug
-  xdebug: 'debug'
+  xdebug: false
```

Comment:
Turned this off by default as it kicks out errors at the end and it runs slightly quicker, probably the 'debug' value isn't quite right anymore.